tensorly/utils/tests/test_prod.py

```diff
@@ -1,10 +0,0 @@
-from .._prod import _prod
-from math import prod
-import tensorly as tl
```

Comment:
I figured, balancing the tradeoffs, that it was better to remove this test but allow testing on Python 3.7. This fallback rarely occurs, has previously been tested, and will be removed soon when 3.7 reaches end-of-life.

Reply:
Makes sense -- can we instead just comment it out (or better yet, skip for Python >= 3.8.x) and make a note to remove utils.prod altogether when we drop Python 3.7 support?